package.json

```diff
@@ -15,6 +15,9 @@
     "readme.md",
     "vanilla"
   ],
+  "bin": {
+    "maybe-pod-install": "bin/maybe-pod-install"
```

Comment:
Do we need to do `./bin/maybe-pod-install` to be slightly safer?

Reply:
Nope, here's why: The `maybe-pod-install` script is part of the `ignite-bowser` package; it's not installed in the target app at all, but since the target app has the boilerplate package as a dependency, (and here's where I'm handwaving a little because I don't understand the mechanism) node takes care of making sure executables provided by dependency packages are in the path. Somehow.

 At the point where yarn is running (whether now, or later whenever the developer runs it), the current directory is the target app, so I'm depending on that mechanism to locate `maybe-pod-install`, just like it locates `solidarity` and `jetify` already.

(I'm assuming something's wrong with this mechanism in `npm` when the dependency is file-system-relative, as it is when we use the `-b` option with a path to the boilerplate, but `yarn` seems to get it right, and since we strongly recommend `yarn`, I'm not going to dig further now.)